cdk/lib/support-reminders.ts

```diff
@@ -57,6 +57,20 @@ export class SupportReminders extends GuStack {
 			retentionPeriod: Duration.days(14),
 		});
 
+		new GuAlarm(this, `${app}-alarm`, {
```

Comment:
this means we'll get an alarm when signups events fail

Reply:
This was going to be a question of mine!